src/commands/utility/docs.js

```diff
@@ -0,0 +1,24 @@
+const { Command } = require('../../')
+const fetch = require('node-fetch')
+
+module.exports = class Docs extends Command {
+  constructor (client) {
+    super({
+      name: 'docs',
```

Comment:
Agreed. I'd say use "discordjs" as the main name and have a few derivatives from it, like "djsdocs", etc. Never just "docs", because we'll surely have docs commands for other things in the future.